packages/react-instantsearch-dom/src/components/Stats.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { translatable } from 'react-instantsearch-core';
+import { createClassNames } from '../core/utils';
+
+const cx = createClassNames('Stats');
+
+export type StatsComponent = {
+  className?: string;
+  isSmartSorted: boolean;
+  nbHits: number;
+  nbSortedHits?: number;
+  processingTimeMS: number;
+  translate(
+    key: string,
+    n: number,
+    ms: number,
+    nSorted?: number,
+    isSmartSorted?: boolean
+  ): string;
+};
+
+const Stats: React.FC<StatsComponent> = ({
+  className = '',
+  isSmartSorted,
+  nbHits,
+  nbSortedHits,
+  processingTimeMS,
+  translate,
+}) => {
+  return (
+    <div className={classNames(cx(''), className)}>
+      <span className={cx('text')}>
+        {translate(
+          'stats',
+          nbHits,
+          processingTimeMS,
+          nbSortedHits,
+          isSmartSorted
+        )}
+      </span>
+    </div>
+  );
+};
+
+Stats.propTypes = {
+  className: PropTypes.string,
+  isSmartSorted: PropTypes.bool.isRequired,
+  nbHits: PropTypes.number.isRequired,
+  nbSortedHits: PropTypes.number,
+  processingTimeMS: PropTypes.number.isRequired,
+  translate: PropTypes.func.isRequired,
+};
+
+export default translatable({
+  stats: (
+    n: number,
+    ms: number,
```

Comment:
I only just realised ms is a number and not the ms module, it makes sense like this!